docs/features.rst

```diff
@@ -17,11 +17,14 @@ Storing features
 ****************
 
 Features in Lhotse are stored as numpy matrices with shape ``(num_frames, num_features)``.
-By default, we use `lilcom`_ for lossy compression and reduce the size on the disk approximately by half.
+By default, we use `lilcom`_ for lossy compression and reduce the size on the disk by about 3x.
 The lilcom compression method uses a fixed precision that doesn't depend on the magnitude of the thing being compressed, so it's better suited to log-energy features than energy features.
-For now, we extract the features for the whole recordings, and store them in separate files.
-We retrieve them by loading the whole feature matrix and selecting the relevant region (e.g. specified by a cut).
-Eventually we will look into optimizing the storage to further reduce the I/O.
+We currently support two kinds of storage:
+
+- HDF5 files with multiple feature matrices (recommended)
```

Comment:
Will this actually give a speed boost?  I would imagine it depends on the order of access?

Reply:
I've just ran some tests with random access, it does seem to take 50-70% longer time to pass over mini_librispeech features in HDF5 than sequential access. But then sometimes it's the same speed if you run it many times (caching kicks in) and (probably) nobody else is using that disk. So I'm not sure how much slowdown is due to random access and how much due to other users. But for sure there is an opportunity to optimize it further with some kind of chunked storage etc.

Reply:
I removed the recommendations - we obviously need to spend more time playing with this